### PR TITLE
#433: [Panel] should be clickable anywhere in collapsed state (closes #433)

### LINKED ISSUE #433: [Panel] should be clickable anywhere in collapsed state
## description
- as for title, collapsible vertical panels in collapsed state have a small clickable area (the header) to expand them
- make all the panel area clickable (i.e. add onClick handler to panel-content in collapsed state)

### PR DESCRIPTION
**Pull request by francescogior**
Issue #433

### CODE REVIEW

**Comment by FrancescoCioria** [comment]
test plan is missing (also have a look at the comment I left on the issue)

PS: why isn't there any CSS fix? (I expected at least a `cursor: pointer`) did you discuss this with alessandra? `Panel` is a style-guideline component, it should always be in sync with its guidelines

**Comment by francescogior** [comment]
@FrancescoCioria updated test plan
- there is a `cursor: pointer` in the new clickable area
- there is also an 'hover effect' as when the mouse in the panel content, the `expand-icon` changes color

**Comment by francescogior** [comment]
@FrancescoCioria ^

**Comment by FrancescoCioria** [comment]
@francescogior I fixed a regression in the css: the icon wasn't changing on `hover` when expanded.

Have a look, if it looks fine to you we can merge it

File: src/Panel/panel.scss [hunk 1]
```diff
@@ -42,30 +42,33 @@ $content-title-color: #2c3a40 !default;
...
+
+        .expand-icon {
+          margin-right: 3px;
+        }
+
+        &:hover .expand-icon {
+          color: $expand-icon-color-hover;
+        }
       }
     }
```
Comment (francescogior) [inline src/Panel/panel.scss]: great much better!

**Comment by francescogior** [comment]
perfect!!

**Pull request merged**